### DISCUSSION TITLE
About mHM soil moisture simulation

### DISCUSSION THREAD

**Discussion by klekarkar** [mhm-ufz/mHM#113] [Q&A] [answered]
Dear mHM community,
I am using mHM for simulating long-term soil moisture conditions. While at it, I noticed that the model consistently gives higher than expected volumetric soil moisture for all the layers. I attach the figure below comparing mHM and GLEAM soil moisture averaged over a 1.5x1.5 degree region. I find a pretty good temporal match, but mHM gives values closer to 1 compared to a max value of about 0.36 from GLEAM during the wettest periods. Is the difference caused by the way mHM calculates SM? I checked the `mo_soil_moisture.90` file and found that mHM calculates $\theta$ relative to saturation and not field capacity. Do I understand it correctly that the volumetric soil moisture content of mHM (SM_L0x) is from this equation?
theta_inorm = (soil_moist - wilting_point) / (soil_moist_sat - wilting_point) 

and not the conventional way of limiting sm to field capacity?

For clarity GLEAM soil moisture is computed by 

$\theta = \frac{soil moisture (mm)}{\Delta z} \leq \theta_{\text{fc}} $

I appreciate your feedback in advance

![image](https://github.com/user-attachments/assets/b48ccb82-6dcb-46df-9ae7-9c67e2d8be00)

**Comment by StephanThober** [comment, marked as answer]
Dear klekarkar,

In mHM, volumetric soil moisture in mHM is caluclated as 

SM = soil_moist / soil_moist_sat

which implies that the lowest value is not 0, but wilting_point / soil_moist_sat .

I do not think that soil moisture is usually limited by field capacity. Soil moisture often exceeds field capacity after heavy rainfall events. The difference between the two equations you use here is that the denominator for GLEAM is the soil depth, so the maximum value for GLEAM is porosity / soil_depth, which will be between 0.36 to 0.45 depending on soil texture. It would be interesting to see the plot if you align the normalization.

**Reply by klekarkar** [reply]
Thank you @StephanThober,
I understand this very well.
I was struggling to explain why my simulated SWC_L values are higher in soils with higher sand content, but I think this explains it.
